samples/nrf_desktop/src/events/config_event.h

```diff
@@ -34,20 +35,52 @@ enum config_event_id {
 	CONFIG_EVENT_ID_MAX
 };
 
-
 /** @brief Configuration channel event.
  * Used to change firmware parameters at runtime.
  */
 struct config_event {
 	struct event_header header;
 
 	enum config_event_id id;
-	u8_t data[REPORT_SIZE_USER_CONFIG - 1];
+	u8_t data[DATA_SIZE_USER_CONFIG];
 	bool store_needed;
 };
 
 EVENT_TYPE_DECLARE(config_event);
 
+
+/** @brief Configuration channel forward event.
+ * Used to pass configuration from dongle to connected devices.
+ */
+struct config_forward_event {
+	struct event_header header;
+
+	u16_t recipient;
+	enum config_event_id id;
+	u8_t data[DATA_SIZE_USER_CONFIG];
+};
+
+EVENT_TYPE_DECLARE(config_forward_event);
+
+
+enum config_forwarded_status {
```

Comment:
good practice is to keep naming within enum consistent, also I am a bit worried about the `CONFIG_` prefix which is used already by Kconfig. In this case it would be:
```
enum forward_status {
   FORWARD_STATUS_PENDING,
   FORWARD_STATUS_SUCCESS,
...
};
```

Reply:
Good idea, done